cmd/swarm/flags.go

```diff
@@ -65,6 +65,16 @@ var (
 		Usage:  "URL of the Ethereum API provider to use to settle SWAP payments",
 		EnvVar: SwarmEnvSwapBackendURL,
 	}
+	SwarmSwapPaymentThresholdFlag = cli.Uint64Flag{
+		Name:   "swap-payment-threshold",
+		Usage:  "honey amount indebted to a peer at which you will initiate payment",
+		EnvVar: SwarmEnvSwapPaymentThreshold,
+	}
+	SwarmSwapDisconnectThresholdFlag = cli.Uint64Flag{
+		Name:   "swap-disconnect-threshold",
+		Usage:  "honey amount at debt of a peer at which you will disconnect",
```

Comment:
And here "honey amount at which a peers disconnect"

Reply:
I would like to keep this from the point of view of the node operator

Reply:
Same as above